rc/filetype/javascript.kak

```diff
@@ -99,7 +99,7 @@ define-command -hidden javascript-insert-on-new-line %<
         # trim trailing whitespace on the previous line
         try %[ execute-keys -draft s\h+$<ret> d ]
         # align the new star with the previous one
-        execute-keys Kx1s^[^*]*(\*)<ret>&
+        execute-keys Kx1s^[^*]*(\*)<ret><a-(><a-&>
```

Comment:
Nice find, LGTM. 
To reproduce it's enough to press `o` on a file

		/*

	         */

I wonder how many of these incorrect uses of & are hidden in rc/filetype/.
I personally have all the auto-insert hooks turned off.